### PR TITLE
nvidia-display-driver: add recipe

### PR DESCRIPTION
**Pull request by ichergui**
This add display driver support for tegra234

Signed-off-by: Ilies CHERGUI <ilies.chergui@gmail.com>

### CODE REVIEW

File: recipes-bsp/tegra-binaries/tegra-configs_34.1.0.bb [hunk 1]
```diff
@@ -48,11 +48,13 @@ do_install:append:tegra194() {
...
 do_install:append:tegra234() {
     install -m 0644 ${S}/etc/X11/xorg.conf ${D}${sysconfdir}/X11/xorg.conf
+    install -d ${D}${sysconfdir}/X11/xorg.conf.d
+    install -m 0644 ${S}/etc/X11/xorg.conf.d/tegra-drm-outputclass.conf ${D}${sysconfdir}/X11/xorg.conf.d
 }
 
 PACKAGES = "${PN}-udev ${PN}-omx-tegra ${PN}-xorg ${PN}-nvstartup ${PN}-container-csv"
 FILES:${PN}-udev = "${sysconfdir}/udev/rules.d"
-FILES:${PN}-xorg = "${sysconfdir}/X11"
+FILES:${PN}-xorg = "${sysconfdir}/X11 ${sysconfdir}/X11/xorg.conf.d"
```
Comment (madisongh) [inline recipes-bsp/tegra-binaries/tegra-configs_34.1.0.bb:57]: This addition wasn't strictly necessary, but that's OK

File: recipes-kernel/nvidia-display-driver/nvidia-display-driver_34.1.0.bb [hunk 2]
```diff
@@ -0,0 +1,40 @@
+SUMMARY = "Linux kernel modules for Nvidia Display"
+DESCRIPTION = "${SUMMARY}"
+LICENSE = "Proprietary"
```
Comment (madisongh) [inline recipes-kernel/nvidia-display-driver/nvidia-display-driver_34.1.0.bb:3]: The license described in the COPYING file is `MIT | GPLv2`

**Review by madisongh — CHANGES_REQUESTED** [review]
One minor change, otherwise LGTM

File: recipes-bsp/tegra-binaries/tegra-configs_34.1.0.bb (hunk 1 above)
Comment (ichergui) [reply recipes-bsp/tegra-binaries/tegra-configs_34.1.0.bb:57]: I will remove it

File: recipes-kernel/nvidia-display-driver/nvidia-display-driver_34.1.0.bb (hunk 2 above)
Comment (ichergui) [reply recipes-kernel/nvidia-display-driver/nvidia-display-driver_34.1.0.bb:3]: I will update the `LICENSE` list

**Review by madisongh — APPROVED** [review]

**Pull request merged**